tests/checker/good/gold/adt_test.scilla.gold

```diff
@@ -15,14 +15,14 @@
     "events": [],
     "ADTs": [
       {
-        "tname": "TestType1",
+        "tname": "adt_test.scilla.TestType1",
```

Comment:
Should we use the unqualified name here? I think the "ADTs" group is used by savant-IDE, so we should provide something that is easy to use for IDEs. Also, since this is output from `scilla-checker` we can't assume we have an address to output at this point, since the contract won't have been deployed yet.

Come to think of it, though, maybe it is better to use the qualified name (using the library name as the qualifier), since it makes it easy to distinguish between user-defined and built-in ADTs.

I'm still not certain, though, so alternative opinions are welcome. :-)